nlu.py

```diff
@@ -20,6 +20,34 @@ def _get_commands(text_sequence: [str]) -> set:
     return possible_commands
 
 
+def _consine_similarity(question: [str], text_sequence: [str]) -> float:
```

Comment:
`question` and `text_sequence` seems more like `question_pattern` and `what_user_asked`, am I correct? But later you would use this fundamental function, I would say `a`, `b` does better name actually because this is a general purpose function.

Reply:
you're right called a, b is more clear and understanding.